src/main/java/uk/gov/hmcts/reform/sscs/personalisation/Personalisation.java

```diff
@@ -221,7 +221,8 @@ void setHearingContactDate(Map<String, String> personalisation, SscsCaseDataWrap
             for (Event event : ccdResponse.getEvents()) {
                 if ((event.getValue() != null)
                     && ((notificationEventType.equals(APPEAL_RECEIVED_NOTIFICATION) && event.getValue().getEventType().equals(APPEAL_RECEIVED)))
```

Comment:
This condition is becoming complicated.  Can you refactor it and give it a sensible name so that the condition looks clear in the `if` statement?